src/notation/view/searchpopupmodel.cpp

```diff
@@ -32,14 +32,22 @@ void SearchPopupModel::load()
     });
 }
 
-void SearchPopupModel::search(const QString& text)
+bool SearchPopupModel::search(const QString& text)
 {
     mu::engraving::EngravingItem* element = notation()->elements()->search(text.toStdString());
     if (element) {
+        if (element->isPage()) {
```

Comment:
Should rehearsal marks also have the same behavior of selecting the rehearsal marks they're attached to?

Reply:
Any reason why they shouldn't?